### PR TITLE
[IOS-299]: Get rid of blink while showing splash

### PR DESCRIPTION
**Pull request by yuliaveres**

### CODE REVIEW

File: Toshi/Controllers/Sign In/SplashViewController.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ final class SplashViewController: UIViewController {
         let imageView = UIImageView(withAutoLayout: true)
         imageView.contentMode = .scaleAspectFill
         imageView.isUserInteractionEnabled = true
-        imageView.image = UIImage(named: "splash")
+        imageView.image = UIImage(named: "launch-screen")
```
Comment (roberthein) [inline Toshi/Controllers/Sign In/SplashViewController.swift]: 💥

File: Toshi/Controllers/Sign In/SplashViewController.swift [hunk 2]
```diff
@@ -97,14 +97,6 @@ final class SplashViewController: UIViewController {
         decorateView()
     }
 
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
-
-        UIView.animate(withDuration: 0.2) {
-            self.view.alpha = 1.0
```
Comment (roberthein) [inline Toshi/Controllers/Sign In/SplashViewController.swift]: Why did we have this?

File: Toshi/Controllers/TabBarController.swift [hunk 3]
```diff
@@ -73,10 +73,7 @@ open class TabBarController: UITabBarController {
         fatalError()
     }
 
-    open override func viewDidLoad() {
-        super.viewDidLoad()
-
-        // TODO: Refactor all this navigation controllers subclasses into one, they have similar code
+    public func setupControllers() {
```
Comment (roberthein) [inline Toshi/Controllers/TabBarController.swift]: Better. 👍

File: Toshi/Controllers/TabBarController.swift [hunk 4]
```diff
@@ -101,7 +98,6 @@ open class TabBarController: UITabBarController {
 
         view.tintColor = Theme.tintColor
 
-        view.backgroundColor = Theme.viewBackgroundColor
```
Comment (roberthein) [inline Toshi/Controllers/TabBarController.swift]: Should this be removed?

File: Toshi/Resources/Assets.xcassets/Splash/logo.imageset/Contents.json [hunk 5]
```diff
@@ -2,17 +2,17 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "logo.pdf",
+      "filename" : "logo@1x.png",
       "scale" : "1x"
```
Comment (roberthein) [inline Toshi/Resources/Assets.xcassets/Splash/logo.imageset/Contents.json]: 🙏 🙏 🙏

File: Toshi/Supporting Files/AppDelegate.m [hunk 6]
```diff
@@ -201,8 +208,15 @@ - (void)didCreateUser {
 
 - (void)presentSplash
 {
+    UIImageView *imageView = [[UIImageView alloc] initWithImage:[UIImage imageNamed: @"launch-screen"]];
+    imageView.contentMode = UIViewContentModeScaleAspectFill;
+    imageView.userInteractionEnabled = YES;
+    [self.window addSubview:imageView];
```
Comment (roberthein) [inline Toshi/Supporting Files/AppDelegate.m]: 🤷‍♂️

File: Toshi/Controllers/Sign In/SplashViewController.swift (hunk 2 above)
Comment (yuliaveres) [reply Toshi/Controllers/Sign In/SplashViewController.swift]: i dnt remember

File: Toshi/Controllers/TabBarController.swift (hunk 4 above)
Comment (yuliaveres) [reply Toshi/Controllers/TabBarController.swift]: now it doesnt make any change to the fix. so i will put it back

**Pull request merged**